test/test_util/prop_utils.dart

```diff
@@ -14,35 +14,13 @@
 
 library test_util.prop_utils;
 
-// Tell dart2js that this library only needs to reflect types annotated with `Props`/`PropsMixin`.
-// This speeds up compilation and makes JS output much smaller.
-@MirrorsUsed(metaTargets: const [
-  'over_react.component_declaration.annotations.Props',
-  'over_react.component_declaration.annotations.PropsMixin',
-])
-import 'dart:mirrors';
-
 import 'package:over_react_test/over_react_test.dart';
 import 'package:test/test.dart';
 
-void testProp(Symbol name, dynamic expectedKey, instance, testValue) {
-  InstanceMirror mirror = reflect(instance);
-
-  mirror.setField(name, testValue);
-  expect(instance[expectedKey], equals(testValue));
-  expect(mirror.getField(name).reflectee, equals(testValue));
-}
-
-void testKeys(List<String> keys, dynamic instanceBuilder()) {
+void testInvalidKey(dynamic instanceBuilder()) {
   test('cannot set / read values that are not its prop map', () {
     var instance = instanceBuilder();
     expect(() {instance['notThere'];},
         throwsA(hasToStringValue(contains('Map does not contain this key'))));
   });
-  for (var propKey in keys) {
-    test('prop: $propKey can have its value set / read', () {
```

Comment:
These tests were only intended to verify that the code generation for implementing concrete getters/setters was working properly, and that these props classes had a prop key namespace of `''`, (except for the case of aria props, which are prefixed with `aria-`). Therefore, these tests have been simplified [here](https://github.com/Workiva/over_react/pull/219/files#diff-33b0e4fc8ff490302629b793a0245148R26) to just test on a few of the props on a mixin.